docs/changelog.txt

```diff
@@ -40,6 +40,8 @@ changelog.txt uses a syntax similar to RST, with a few special sequences:
 - Fix right click sometimes closing both a DFHack window and a vanilla panel
 - Fixed issue with scrollable lists having some data off-screen if they were scrolled before being made visible
 - `channel-safely`: fixed bug resulting in marker mode never being set for any designation
+- `automelt`: fixed bug related to lua stack smashing behavior in returned stockpile configs
+- `autochop`: fixed bug related to lua stack smashing behavior in returned stockpile configs
```

Comment:
For reference (I will fix): these should be marked as dev-only changes with the `@` prefix, because they aren't relevant to users of the stable channel who would have never seen the issue before it was fixed.